tools/src/main/java/com/nedap/archie/archetypevalidator/ArchetypeValidator.java

```diff
@@ -179,7 +179,7 @@ public ValidationResult validate(Archetype archetype, FullArchetypeRepository re
                 } catch (Exception e) {
                     //this is probably an error in an included archetype, so ignore it here
                     //the other archetype will not validate
-                    messages.add(new ValidationMessage(ErrorType.OTHER, "Validation failed with exception " + e));
+                    messages.add(new ValidationMessage(ErrorType.VUNP, "Validation failed with exception " + e));
```

Comment:
This is not a VUNP, it has little to do with use_node.  It is something along the lines of
```java
ValidationMessage message = new ValidationMessage(ErrorType.OTHER, "Error during Operational template creation. This does not necessarily mean the current archetype has a problem, but perhaps one that is included with use_archetype: " + e);
message.setWarning(true);
messages.add(message);
```

Or even better, a better constructor that allows to set the message type (but perhaps not in this PR)